### PR TITLE
Shortlist button focusable

### PR DESCRIPTION
**Pull request by lucascumsille**
Bug found while working on this PR: https://github.com/mysociety/fixmystreet/pull/5208

This PR changes the markup of `label.btn--shortlist` and swaps it for `button.btn--shortlist`, that way the element is focusable and can also be interacted via keyboard.

[Skip changelog]

### CODE REVIEW

**Review by dracos — COMMENTED** [review]
This works, but I wonder if there's an easier solution - if we switch this label to be an actual button with `form="planned_form"` (and adjustments accordingly) then I don't think the submit button in the planned_form is even needed - perhaps whoever did this originally didn't know buttons in a form can actually be linked to a different form?

**Comment by lucascumsille** [comment]
> This works, but I wonder if there's an easier solution - if we switch this label to be an actual button with `form="planned_form"` (and adjustments accordingly) then I don't think the submit button in the planned_form is even needed - perhaps whoever did this originally didn't know buttons in a form can actually be linked to a different form?

Thanks for the feedback, I thought there would be a more powerful reason to use input and label. So I swapped it for a button and it seems to be working =)

File: web/cobrands/fixmystreet/fixmystreet.js [hunk 1]
```diff
@@ -509,38 +509,40 @@ $.extend(fixmystreet.set_up, {
...
+            var inputName, newInputName, buttonLabel, buttonValue, classToAdd, classToRemove;
+
+            if (data.outcome === 'add') {
+                inputName = 'shortlist-add';
+                newInputName = 'shortlist-remove';
                 buttonLabel = $submit.data('label-remove');
                 buttonValue = $submit.data('value-remove');
                 classToAdd = $submit.data('class-remove');
                 classToRemove = $submit.data('class-add');
-                $('.shortlisted-status').remove();
```
[the remark this answers is not in the document]
Comment (lucascumsille) [reply web/cobrands/fixmystreet/fixmystreet.js:532]: @dracos my bad I thought it wasn't doing anything anymore. I added a fixup to put it back

**Review by dracos — APPROVED** [review]
Great :)

**Pull request merged**